docs/resources/vod_transcoding_template_group.md

```diff
@@ -0,0 +1,145 @@
+---
+subcategory: "Video on Demand (VOD)"
+---
+
+# huaweicloud_vod_media_category
```

Comment:
s/media_category/template_group

Reply:
not `template_group`, is `transcoding_template_group`.

Reply:
not `template_group`, is `transcoding_template_group`.

Reply:
fixed